nifi-nar-bundles/pom.xml

```diff
@@ -94,7 +95,7 @@
     <build>
         <plugins>
             <!-- Makes buildRevision and buildBranch available to the NAR Plugin so they can be populated in the MANIFEST -->
-            <plugin>
+            <!--plugin>
```

Comment:
If this is not needed it should be removed

Reply:
fixed (uncommented)